yt_dlp/extractor/crunchyroll.py

```diff
@@ -720,15 +720,27 @@ class CrunchyrollBetaBaseIE(CrunchyrollBaseIE):
 
     def _get_params(self, lang):
         if not CrunchyrollBetaBaseIE.params:
+            if self._get_cookies(f'https://beta.crunchyroll.com/{lang}').get('etp_rt'):
+                auth = 'cookie'
+            else:
+                auth = 'client id'
```

Comment:
```suggestion
            if self._get_cookies(f'https://beta.crunchyroll.com/{lang}').get('etp_rt'):
                grant_type, key = 'etp_rt_cookie', 'accountAuthClientId'
            else:
                grant_type, key = 'client_id', 'anonClientId'
```
should make it  much simpler (see comment below)

Reply:
`auth` is also used in the `note` for `auth_response`. I suppose I can just add it to the list of variables assigned, though.

Reply:
Just use `grant_type`

Reply:
Hmm, ok, I suppose.